README.md

```diff
@@ -1 +1,30 @@
 # Uno Chefs
```

Comment:
@kazo0 do we wanna use that **Recipe Books** on README.md as well?

If so, we could have an aka link for that so it works from the README.md as well as the docs.

cc @agneszitte

Reply:
For now we can merge without and add later, as long as there's some sort of content to display its fine